erpnext/accounts/report/general_ledger/general_ledger.py

```diff
@@ -205,10 +209,18 @@ def get_gl_entries(filters, accounting_dimensions):
 
 def get_conditions(filters):
 	conditions = []
+
 	if filters.get("account") and not filters.get("include_dimensions"):
-		lft, rgt = frappe.db.get_value("Account", filters["account"], ["lft", "rgt"])
-		conditions.append("""account in (select name from tabAccount
-			where lft>=%s and rgt<=%s and docstatus<2)""" % (lft, rgt))
+		account_conditions = ""
```

Comment:
Instead of adding multiple "OR" conditions one SET of accounts and can be made and just one single condition for accounts should be passed

Reply:
Could you give it a look now? It's not exactly what you're asking for, but I've reduced it to a single condition.